dev/Scroller/Scroller.cpp

```diff
@@ -1504,8 +1523,8 @@ void Scroller::ComputeBringIntoViewTargetOffsets(
     if (snapPointsMode == winrt::SnapPointsMode::Default)
     {
         // Finally adjust the target offsets based on snap points
-        targetZoomedHorizontalOffsetTmp = ComputeValueAfterSnapPoints(targetZoomedHorizontalOffsetTmp, m_sortedConsolidatedHorizontalSnapPoints);
-        targetZoomedHorizontalOffsetTmp = ComputeValueAfterSnapPoints(targetZoomedHorizontalOffsetTmp, m_sortedConsolidatedVerticalSnapPoints);
+        targetZoomedHorizontalOffsetTmp = ComputeValueAfterScrollSnapPoints(targetZoomedHorizontalOffsetTmp, m_sortedConsolidatedHorizontalSnapPoints);
+        targetZoomedHorizontalOffsetTmp = ComputeValueAfterScrollSnapPoints(targetZoomedHorizontalOffsetTmp, m_sortedConsolidatedVerticalSnapPoints);
```

Comment:
>targetZoomedHorizontalOffsetTmp = ComputeValueAfterScrollSnapPoints(targetZoomedHorizontalOffsetTmp, m_sortedConsolidatedVerticalSnapPoints) [](start = 8, length = 140)

Is this a bug that was present before this change? Looks like we are chaning the HorizontalOffset with the VerticalSnap Points.

Reply:
That was a typo. Thanks.